### ISSUE TITLE
Parameters for pretrained datasets

### ISSUE THREAD

**Issue by erlebach** [flairNLP/flair#1943]
HI, 

Where can I find the parameters you used to generate the various pre-trained datasets? I tried to duplicate the POS tagging for the OntoNotes dataset, but only got 60% testing accuracy after 100 epochs. This leads me to believe that I am not using a correct set of hyperparameters. If you could provide this information, I would greatly appreciate it. Thank you. Gordon.

**Comment by stefan-it** [comment]
Hi @erlebach ,

could you specify which model you would like to replicate? E.g. OntoNotes results are reported for NER, PoS tagging model is available on Wall Street Journal dataset. The parameters should be documented here:

https://github.com/flairNLP/flair/blob/master/resources/docs/EXPERIMENTS.md

Would be great if you could also provide a code snippet that you're using for training; I can have a look at it then :hugs:

**Comment by cpuyyp** [comment]
Hi @stefan-it ,

I'd like to know the parameters and replicate the fine-grained PoS tagging on OntoNotes dataset, which is reported with a 98.19% accuracy. I found it under the English-models section in the list of pre-trained models. Link here: https://github.com/flairNLP/flair/blob/master/resources/docs/TUTORIAL_2_TAGGING.md#english-models

Really appreciate your help!

**Comment by alanakbik** [comment]
The English Ontonotes POS model shipped in Flair was trained with these embeddings: 

```python
        embeddings = StackedEmbeddings(
            [
                FlairEmbeddings("news-forward", with_whitespace=False),
                FlairEmbeddings("news-backward", with_whitespace=False),
            ]
        )
```

The rest is the same as the other scripts.
